service/controller/chart/resource/status/create.go

```diff
@@ -115,3 +126,39 @@ func (r *Resource) setStatus(ctx context.Context, cr v1alpha1.Chart, status v1al
 
 	return nil
 }
+
+func updateEvent(webhookURL string, status v1alpha1.ChartStatus) error {
+	request := Request{
+		AppVersion:   status.AppVersion,
+		LastDeployed: status.Release.LastDeployed,
+		Reason:       status.Reason,
+		Status:       status.Release.Status,
+		Version:      status.Version,
+	}
+
+	payload, err := json.Marshal(request)
+	if err != nil {
+		return microerror.Mask(err)
+	}
+
+	client := &http.Client{}
```

Comment:
I have a worry here if calling the webhook takes a long time. As this could block reconciling other CRs.

WDYT to using a short timeout here? If the webhook times out we will check again after the resync.

Reply:
I would suggest using the same amount of time for pulling charts. WDYT?

Reply:
Yes good idea. 👍